sys-libs/libnbd/libnbd-1.14.2.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{8..11} )
+
+inherit autotools bash-completion-r1 python-single-r1
```

Comment:
Need to define pkg_setup: https://projects.gentoo.org/python/guide/single.html#conditional-python-use.

Reply:
fix, pkgcheck did not show me any warning